CONTRIBUTING.md

```diff
@@ -59,4 +59,219 @@ These are automatically executed by the `make distcheck`, which will package a n
 spin up the various operating systems, install the newly packaged archive, and execute
 the test suite.
 
+## Releasing
 
+The follow steps outline the release process for a maintenance branch (e.g.
+releasing the `vX.Y` branch as X.Y.Z).
+
+### Ensure PHP version compatibility
+
+Ensure that the extension compiles on PHP 5.4, 5.5, 5.6, and 7.0. Be sure to
+test both ZTS and non-ZTS builds for PHP 5.x.
+
+### Ensure Windows compatibility
+
+PECL will create Windows DLLs for new releases; however, you must ensure that
+the extension successfully builds on Windows before releasing. Note that PHP 5.5
+and 5.6 require VS2012, while PHP 7 requires VS2015.
+
+Given the following assumptions:
+
+ * Build directory is `C:\php-sdk\`
+ * Compiling for PHP 5.6 (VS2012 x86 Native Tools Command Prompt is running)
+ * Extension branch checked out in `C:\php-sdk\phpdev\vc11\x86\pecl\mongodb`
+
+The build process will resemble:
+
+```
+cd c:\php-sdk\
+bin\phpsdk_setvars.bat
+
+cd C:\php-sdk\phpdev\vc11\x86\php-5.6.12-src
+nmake clean
+buildconf --force
+configure --disable-all --with-openssl --enable-cli --enable-json --enable-mongodb
```

Comment:
If you have instructions for doing so, I'd be happy to add them. It isn't covered in the wiki docs, AFAIK.